go.mod

```diff
@@ -151,7 +151,6 @@ require (
 )
 
 replace (
-	github.com/cosmos/cosmos-sdk => github.com/cosmos/cosmos-sdk v0.45.15-ics
```

Comment:
This replace doesn't do anything now that we reference mainline sdk and not a special informal fork

Reply:
cool

Reply:
wait Khanh and I actually do know how to use Git?

:/